.github/scripts/update-sources.sh

```diff
@@ -53,19 +55,28 @@ function update_repository() {
             git submodule update
         popd
     fi
+
+    pushd $DIRECTORY
+        if [[ "$REBASE_SOURCES" = 1 && -n "$DEVEL_BRANCH" ]]; then
+            git rebase origin/$DEVEL_BRANCH
```

Comment:
You are right I have it alredy fixed in other branch so that rebase will be done only when FLAT\_CLONE=0 in other branch. I'll update it here as well.

Reply:
Fixed by not doing rebase when `FLAT_CLONE=1`.